### ISSUE TITLE
[steamshovel] option dropdown disappears (Trac #2044)

### ISSUE THREAD

**Issue by icecube-trac** [icecube-trac/tix2#2036]
On some of the option dropdowns, if the selection is blank (the default), the dropdown disappears into the white background.  This confuses users, as they don't know where to click.

Migrated from https://code.icecube.wisc.edu/ticket/2044
```json
{
    "status": "closed", 
    "changetime": "2019-02-13T14:14:03", 
    "description": "On some of the option dropdowns, if the selection is blank (the default), the dropdown disappears into the white background.  This confuses users, as they don't know where to click.", 
    "reporter": "david.schultz", 
    "cc": "", 
    "resolution": "fixed", 
    "_ts": "1550067243750755", 
    "component": "combo core", 
    "summary": "[steamshovel] option dropdown disappears", 
    "priority": "major", 
    "keywords": "", 
    "time": "2017-06-23T20:39:06", 
    "milestone": "", 
    "owner": "sander.vanheule", 
    "type": "defect"
}
```

**Comment by icecube-trac** [comment]
Trac update at 20170628T17:10:49: 
* olivas changed status from "new" to "assigned"
* olivas changed owner from "" to "sander.vanheule"

**Comment by icecube-trac** [comment]
Trac update at 20170703T13:50:33: 
* sander.vanheule changed status from "assigned" to "closed"
* sander.vanheule commented:

> In changeset 156905:
> 
> ```text
> #!CommitTicketReference repository="IceCube" revision="156905"
> Display placeholder value for empty KeySetting.
> 
> The placeholder replaces the empty string value that is used to
> indicate that no key is currently selected, but is displayed in
> an italic font to denote the difference from valid values.
> This is the same behaviour as used by an non-selected ArtistKey.
> 
> Fixes #2036.
> ```


* sander.vanheule changed resolution from "" to "fixed"

**Comment by icecube-trac** [comment]
Trac update at 20190213T14:14:03: 
* olivas commented:

> Milestone August 2017 deleted


* olivas changed milestone from "August 2017" to ""
